app/lib/theme.ts

```diff
@@ -1,26 +1,22 @@
 "use client";
-
-import { setCookie } from "./cookieHandlers";
-
-function mediaListner(ev: MediaQueryListEvent) {
-  if (typeof document === "undefined") return;
```

Comment:
This line has to be present otherwise you keep having errors on the server "window is not present" other than that great job! 👍

Reply:
thanks for correction fixing it